app/templates/agenda.html

```diff
@@ -6,51 +6,58 @@
 {% endblock %}
 
 {% block content %}
-  <form method="GET" action="/agenda">
-    <div class="mb-3 agenda_filters">
-      <label for="start_date">{{ gettext("From") }}</label><br>
-      <input class="filter" type="date" id="start_date" name="start_date" value='{{ start_date }}'><br>
-      <label for="end_date">{{ gettext("to") }}</label><br>
-      <input class="filter" type="date" id="end_date" name="end_date" value='{{ end_date }}'><br>
-      <input class="filter" type="submit" value="{{ gettext('Get Agenda') }}">
+  <form method="GET" action="/agenda#dates" class="mx-3 pt-3">
+    <div class="col-sm-3">
+      <label for="start_date">{{ gettext("From") }}</label>
+      <input class="form-control" type="date" id="start_date" name="start_date" value='{{ start_date }}'>
     </div>
-  </form>
-  <div class="exact_date">
-    <div>
-      <a class="btn agenda_button" href="/agenda?days=0">{{ gettext("Today") }}</a>
+    <div class="col-sm-3">
+      <label for="end_date">{{ gettext("To") }}</label>
+      <input class="form-control" type="date" id="end_date" name="end_date" value='{{ end_date }}'><br>
     </div>
     <div>
-      <a class="btn agenda_button" href="/agenda?days=7">{{ gettext("Next week") }}</a>
+      <input class="btn btn-primary btn-sm" type="submit" value="{{ gettext('Get Agenda') }}">
     </div>
-    <div>
-      <a class="btn agenda_button" href="/agenda?days=30">{{ gettext("Next month") }}</a>
+  </form>
+
+  <div class="exact_date pt-3">
+    <div class="mx-3">
+      <a class="btn btn-light btn-outline-primary btn-sm" href="/agenda?days=0#dates">{{ gettext("Today") }}</a>
+    </div>
+    <div class="mx-3">
+      <a class="btn btn-light btn-outline-primary btn-sm" href="/agenda?days=7#dates">{{ gettext("Next 7 days") }}</a>
+    </div>
+    <div class="mx-3">
+      <a class="btn btn-light btn-outline-primary btn-sm" href="/agenda?days=30#dates">{{ gettext("Next 30 days") }}</a>
     </div>
   </div>
 
-
-  <div class="container mt-4">
+  <div class="pt-4 px-5">
     {% if start_date > end_date %}
       <p>{{ gettext("Start date is greater than end date") }}</p>
     {% elif events | length == 0 %}
       <p>{{ gettext("No events found...") }}</p>
     {% elif start_date == end_date %}
-      <h1>{{ start_date.strftime("%d/%m/%Y") }}</h1>
+      <h1 id="dates">{{ start_date.strftime("%d/%m/%Y") }}</h1>
```

Comment:
If we already at it, can you please change it to h2 and have another element with h1? :)

There should be only 1 h1 in page

Reply:
The page will eventually contain only 1 h1 element, as the two h1 elements are part of a condition.
Is it ok or should I change it?
Maybe it's better to use regular <span>, because I don't have another element that can be h1?